test/auth/tokens-manager.tests.js

```diff
@@ -79,7 +80,7 @@ describe('TokensManager', function() {
     it('should return a promise when no callback is provided', function() {
       var returnValue = manager.getInfo('VALID_TOKEN');
 
-      expect(returnValue).to.be.an.instanceOf(Promise);
+      expect(returnValue.then).to.be.a('function');
```

Comment:
`rest-facade` returns a Promise which isn't matching with the Promise in this scope (I tried using different versions of bluebird to coax it but still nothing 😕). Since this is only a check to make sure a promise is returned, the `.then` handler _should_ be fine 🤔

Reply:
rest-facade uses bluebird, that's why they don't have the same type

Reply:
Yeah, I tried leaving bluebird in, and using the same version of bluebird the lib uses, but nothing worked 😔

Given Bluebird can be retired from this library in May, in which this issue will then re-apply, is there any issue with just using this test?

Reply:
Did you try this?

```js
const BlueBirdPromise = require("bluebird");
expect(returnValue).to.be.an.instanceOf(BlueBirdPromise);
```

Reply:
Yep

I'm going to try regenerating the lockfile, since removing and adding the bluebird dep might have shifted the resolved versions.

Reply:
If that doesn't work, you can leave the test checking for `then`

Reply:
Let's see where this goes https://github.com/ngonzalvez/rest-facade/issues/39

Reply:
Got it to work 😎 

I'd be happy to work on the rest-facade issue, although I have my doubts based on commit history and PR activity 😬